targets/ingester/src/transform/agreements.js

```diff
@@ -0,0 +1,167 @@
+import slugify from "@socialgouv/cdtn-slugify";
+import cdtnSources from "@socialgouv/cdtn-sources";
+import remark from "remark";
+import html from "remark-html";
+import find from "unist-util-find";
+import parents from "unist-util-parents";
+
+import { formatIdcc } from "../lib/formatIdcc.js";
+import { getJson } from "../lib/getJson.js";
+
+const compiler = remark().use(html, { sanitize: true });
+/**
+ * @template A
+ * @param {(a:A) => number} fn
+ * @returns {(a:A, b:A) => number}
+ */
+const createSorter = (fn) => (a, b) => fn(a) - fn(b);
+
+const { SOURCES } = cdtnSources;
+
+/**
+ *
+ * @param {string} pkgName
+ * @returns {Promise<ingester.AgreementPage[]>}
+ */
+export default async function getAgreementDocuments(pkgName) {
+  /** @type {import("@socialgouv/kali-data").IndexedAgreement[]} */
+  const agreements = await getJson(`${pkgName}/data/index.json`);
+
+  /** @type {import("@socialgouv/contributions-data").Question[]} */
+  const contributions = await getJson(
+    `@socialgouv/contributions-data/data/contributions.json`
+  );
+
+  /** @type {import("@socialgouv/datafiller-data").AgreementsItem[]} */
+  const agreementsBlocks = await getJson(
+    "@socialgouv/datafiller-data/data/agreements.json"
+  );
+
+  const contributionsWithSlug = contributions.map((contrib) => {
+    const slug = slugify(contrib.title);
+    return {
+      ...contrib,
+      slug,
+    };
+  });
+
+  const agreementPages = [];
+
+  for (const agreement of agreements) {
+    const agreementTree = await getJson(
+      `@socialgouv/kali-data/data/${agreement.id}.json`
+    );
```

Comment:
With the API : 

```js
const kali = require("@socialgouv/kali-data");
kali.getAgreement(agreement.id);
```

Reply:
yup but using the api imply to rebuilt the api each time we want new data (or we could rely on latest version and add a pre script to install dep before running ingest)